### PR TITLE
fix: bootstrapping with lerna

### PR DESCRIPTION
**Pull request by estherthetesteramido**
#### 📲 What

Boostrapping with hoist with lerna to resolve eslinting issues when running install at root.

#### 🤔 Why
		
So we can lint and deploy!
		
#### 🛠 How
		
Using ht with lerna bootstrap, see https://github.com/lerna/lerna/blob/master/doc/hoist.md#lerna-hoisting for limitations.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -10,10 +10,6 @@
     },
     "version": {
       "message": "chore(release): publish"
-    },
-    "bootstrap": {
-      "ignore": "",
-      "npmClientArgs": ["--no-package-lock"]
```
Comment (dnitsch) [inline lerna.json:16]: you can leave this in I think - just remove the `--no-package-lock`
in any case CLI params will take precedence so we can pipe additional custom stuff in from pipeline or elsewhere

File: packages/scaffolding-cli/src/__tests__/domain/workers/integration_testflow.test.ts [hunk 2]
```diff
@@ -33,20 +33,20 @@ describe("mainWorker class", () => {
     const currentDir = process.cwd()
     let tempDir: string
 
-    beforeEach(() => {
+    beforeEach(async () => {
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/src/__tests__/domain/workers/integration_testflow.test.ts:36]: @dnitsch is this working for you? I had issues doing this asynchronously when running from root. The snapshots weren't writing correctly. The generator function would still be running and then the snapshot would have files that were from another test.

File: packages/scaffolding-cli/src/index.ts [hunk 3]
```diff
@@ -47,7 +47,7 @@ async function cliCommand(argv: CliOptions): Promise<ExitMessage>{
 const runOptions = () => {
     yargs
         .scriptName("@amidostacks/scaffolding-cli")
-        .options("c", {
+        .options("config", {
             alias: ["c", "conf"],
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/src/index.ts:51]: If this was failing, then the tests need to be updated to ensure that we capture this in future? @dnitsch

File: packages/scaffolding-cli/src/__tests__/domain/workers/integration_testflow.test.ts (hunk 2 above)
Comment (dnitsch) [reply packages/scaffolding-cli/src/__tests__/domain/workers/integration_testflow.test.ts:36]: yes it is actually. I somehow still remember that it's not meant to, but it is - even in their docs they say beforeEach work the same way as tests...

File: packages/scaffolding-cli/src/index.ts (hunk 3 above)
Comment (dnitsch) [reply packages/scaffolding-cli/src/index.ts:51]: they were failing for me :D that's why I changed it - see the change in prompts.test.ts

**Pull request merged**